iothub_client/samples/iothub_client_device_twin_and_methods_sample/iothub_client_device_twin_and_methods_sample.c

```diff
@@ -315,7 +327,23 @@ static void iothub_client_device_twin_and_methods_sample_run(void)
     }
     else
     {
-        if ((iotHubClientHandle = IoTHubDeviceClient_CreateFromConnectionString(connectionString, protocol)) == NULL)
+#ifdef SAMPLE_USE_CREATE_FROM_CONNECTION_STRING
```

Comment:
>SAMPLE_USE_CREATE_FROM_CONNECTION_STRING [](start = 7, length = 40)

Do we have a customer explicitly and desperately requesting a sample showing this?  

I believe this is only used (or at least required) in practice with AMQP multiplexing which isn't a feature we're pushing.  And we have a sample of this already in samples/iothub_client_sample_amqp_shared_methods.

Reply:
I created it to test the customer scenario. I don't mind to roll it back, but we do not have any other example of twin with multiplex.

---
In reply to: [328723101](https://github.com/Azure/azure-iot-sdk-c/pull/1211#discussion_r328723101) [](ancestors = 328723101)

Reply:
Yes, I think we should roll it back.  For twin/multiplex, per `IoTHubTransport_AMQP_Common_Subscribe_DeviceTwin` 

```c
LogError("Device Twin not supported on device multiplexing scenario");
```

Reply:
In this case, I'll rollback this sample changes. 

---
In reply to: [328730188](https://github.com/Azure/azure-iot-sdk-c/pull/1211#discussion_r328730188) [](ancestors = 328730188)